ci-operator/step-registry/azure/provision/resourcegroup/cross-subscription/azure-provision-resourcegroup-cross-subscription-ref.yaml

```diff
@@ -0,0 +1,25 @@
+ref:
+  as: azure-provision-resourcegroup-cross-subscription
+  from_image:
+    namespace: ocp
+    name: "4.14"
+    tag: upi-installer
+  grace_period: 10m
+  commands: azure-provision-resourcegroup-cross-subscription-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+    - name: USE_HYPERSHIFT_AZURE_CREDS
+      default: "false"
+      documentation: "If true, use hypershift azure credentials"
+    - name: HYPERSHIFT_AZURE_LOCATION
+      default: ""
+      documentation: "The Azure location of the resource group."
+  credentials:
+    - mount_path: /etc/hypershift-ci-jobs-azurecreds
+      name: hypershift-ci-jobs-azurecreds
+      namespace: test-credentials
```

Comment:
```suggestion
```
I do not see these env vars used in the shell script, better to remove it directly?

Reply:
Same as above, updated.